src/scene.ts

```diff
@@ -870,7 +870,8 @@ export class Scene extends AbstractScene implements IAnimatable {
     /** All of the active cameras added to this scene. */
     public activeCameras = new Array<Camera>();
 
-    private _activeCamera: Nullable<Camera>;
+    /** @hidden */
+    public _activeCamera: Nullable<Camera>;

```

Comment:
Why not relying on getActiveCamera()

Reply:
In the component I am doing this._activeCamera = camera._rigCameras[index]; if I use activeCamera instead, a camera changed notification occurs which shouldnt happen when it changes just for rendering.

Reply:
Makes total sense here 👍